tcms/testplans/static/testplans/js/get.js

```diff
@@ -63,5 +64,40 @@ function getTestCaseRowContent(rowContent, testCase) {
     );
     automation_indication_element.removeClass(automated_class_to_remove);
 
+    getTestCaseExpandArea(row, testCase);
+
     return row;
 }
+
+function getTestCaseExpandArea(row, testCase) {
+    row.find('.js-test-case-expand-text').html(testCase.text);
+    if (testCase.notes.trim().length > 0) {
+        row.find('.js-test-case-expand-notes').html(testCase.notes);
+    }
+
+    // draw the attachments
+    var uniqueDivCustomId = `js-tc-id-${testCase.id}-attachments`;
+    // set unique identifier so we know where to draw fetched data
+    row.find('.js-test-case-expand-attachments').parent()[0].id = uniqueDivCustomId;
+
+    jsonRPC('TestCase.list_attachments',[testCase.id],function(data) {
+
+        // cannot use instance of row in the callback
+        var ulElement = $(`#${uniqueDivCustomId} .js-test-case-expand-attachments`);
+
+        if (data.length === 0) {
+            ulElement.children().removeClass('hidden');
+            return;
+        }
```

Comment:
What does this do, I don't understand it?

Reply:
In the template first <li> in the <ul> which hold the attachments is: <li class="list-group-item hidden">{% trans "No attachments" %}</li>. When there is no attachments I remove the hidden class in order to show message that there is no attachments.